docker/vm/nodejs/nginx_vm.conf

```diff
@@ -0,0 +1,84 @@
+upstream django_app {
+  server django:8000 fail_timeout=0;
+}
+
+# Django server
+server {
+  # Listen on port 80 & then redirect to SSL port
+  listen 80;
+  server_name BACKEND;
+
+  # redirection to SSL 
+  location / {
+    return 301 https://$host$request_uri;
+  }
+
+  # Certbot challenge files
+  location /.well-known/acme-challenge/ {
+    root /var/www/certbot;
+  }
+}
+
+server {
+  listen 443 ssl;
+  server_name BACKEND;
+  client_max_body_size 400M;
+
+  # SSL certificates generated by init-letsencrypt
+  # Don't change the FRONTEND. Certbot places certificates in the first domain passed in init-letsencrypt. 
+  # See https://certbot.eff.org/docs/using.html#where-are-my-certificates to know more
+  ssl_certificate /etc/letsencrypt/live/FRONTEND/fullchain.pem;
```

Comment:
this should be `BACKEND`

Reply:
https://certbot.eff.org/docs/using.html#where-are-my-certificates
In the case of creating a SAN certificate with multiple alternative names, $domain is the first domain passed in via -d parameter. Rather than copying, please point your (web) server configuration directly to those files (or create symlinks)

Since Frontend was passed first, all certs are available in frontend

Reply:
wait, but we have two domain names. Are certs for both domains at the same location?

Reply:
Yes. We're using the same certs